### PR TITLE
Update adding-private-network.md

### PR DESCRIPTION
**Pull request by jerry-li-dev**

### CODE REVIEW

File: src/pages/docs/collaborating-in-postman/adding-private-network.md [hunk 1]
```diff
@@ -50,15 +51,15 @@ If you're logged in, go to **Home** in the upper-left of Postman, and click **Pr
 
 > You can only add an API to the network if you have edit access to the API, and your team members have permission to view or edit the API. Learn more about [roles and permissions](/docs/collaborating-in-postman/roles-and-permissions/).
 
-To add an API to your team's Private API Network from [Postman](https://go.postman.co), sign in and click **Private API Network** in the left navigation bar.
+To add an API to your team's Private API Network from [Postman](https://go.postman.co), sign in and select **Private API Network** in the left navigation bar.
 
 <img alt="Private API Network Left Navbar" src="https://assets.postman.com/postman-docs/private-api-network-left-navbar-v8.jpg" height="350px"/>
 
-The default view is your team's [Private API Network](https://go.postman.co/network/private). To add an API to your team's Private API Network, click **+ Add New**.
+The default view is your team's [Private API Network](https://go.postman.co/network/private). To add an API to your team's Private API Network, in left navigation bar, select **+ Add New**.
```
Comment (jkonrath-postman) [inline src/pages/docs/collaborating-in-postman/adding-private-network.md:58]: in left/in the left

File: src/pages/docs/collaborating-in-postman/adding-private-network.md [hunk 2]
```diff
@@ -50,15 +51,15 @@ If you're logged in, go to **Home** in the upper-left of Postman, and click **Pr
...
 
 <img alt="Private API Network" src="https://assets.postman.com/postman-docs/private-api-network-v8.jpg"/>
 
-In the modal, choose the workspace containing the API you want to add to the network. You can select as many APIs to add to the network at one time. For each API selected, specify the versions to add. Click **Next** and select the folder to add the APIs to. Click **Add to Network**.
+In the modal, choose the workspace containing the API you want to add to the network. You can select as many APIs to add to the network at one time. For each API selected, specify the versions to add. Select **Next** and then the folder to add the APIs to. Select **Add to Network**.
```
Comment (jkonrath-postman) [inline src/pages/docs/collaborating-in-postman/adding-private-network.md:62]: "You can select as many APIs to..." should that be "You can select as many APIs as you want to" or maybe just "You can select one or more APIs to" ?

**Review by jkonrath-postman — APPROVED** [review]
A few minor things, but otherwise looks good

File: src/pages/docs/collaborating-in-postman/adding-private-network.md (hunk 2 above)
Comment (jerry-li-dev) [reply src/pages/docs/collaborating-in-postman/adding-private-network.md:62]: Changed to, "You can select one or more APIs to."

**Pull request merged**